### ISSUE TITLE
how can i we animation same animation oppsite on hide?

### ISSUE THREAD

**Issue by kingsanu** [vaibhav111tandon/vov.css#33]
i using fade-in on element via $('#').show() but how same animation with oppsite effect work on $('#').hide() please note to set animation class by javascript is very hard

**Comment by vaibhav111tandon** [comment]
Hi @kingsanu 
Can you please explain a little bit more about what issue are you facing, do you want to use fade-in with show and hide jQuery functions or something else?

**Comment by kingsanu** [comment]
yes so fade-in-up work fine on $('#element').show(), but when i call $('#element').hide() i want oppsite animation on hide element exampe 'fade-out-down' something, so animation work well on show and hide both

**Comment by vaibhav111tandon** [comment]
@kingsanu If i am right your requirement is to have show() and hide() on the same animation without adding or removing any animation class in jQuery?
